jplotter/src/main/java/hageldave/jplotter/canvas/JPlotterCanvas.java

```diff
@@ -233,22 +233,22 @@ public default PDDocument paintPDF() throws IOException {
 	public default void paintPDF(PDDocument document, PDPage page) throws IOException {
 		int w,h;
 		if ((w=asComponent().getWidth()) > 0 && (h=asComponent().getHeight()) > 0) {
+			// do we need the mediabox?
 			page.setMediaBox(new PDRectangle(w, h));
```

Comment:
I made this part call `paintPDF(PDDocument document, PDPage page, Rectangle2D renderLoc)` since it is the same semantic. But the other method is missing the `setMediaBox` call. What is it for? do we need it?

Reply:
The setMediaBox called is necessary to set the size of the pdf page. We can't skip it, as the size gets set to a default "letter" size, which can be assumed to be wrong in nearly every case.